ertviz/controllers/ensemble_selector_controller.py

```diff
@@ -99,18 +103,28 @@ def update_ensemble_selection(selected_nodes):
             Output(parent.uuid("ensemble-selector"), "className"),
             Output(parent.uuid("ensemble-selector-container"), "className"),
             Output(parent.uuid("ensemble-selector-button"), "children"),
+            Output(parent.uuid("ensemble-view-store"), "data"),
         ],
         [
             Input(parent.uuid("ensemble-selector-button"), "n_clicks"),
         ],
         [
             State(parent.uuid("ensemble-selector"), "className"),
             State(parent.uuid("ensemble-selector-container"), "className"),
+            State(parent.uuid("ensemble-view-store"), "data"),
         ],
     )
-    def update_ensemble_selector_view_size(n_clicks, class_name, class_name_container):
-        n_clicks = 0 if n_clicks is None else n_clicks
-        if n_clicks % 2 == 0:
+    def update_ensemble_selector_view_size(
+        n_clicks, class_name, class_name_container, maximized
```

Comment:
Looks like n_clicks is not in use anymore, maybe it can be replaced by underscore?

Reply:
👍